index.html

```diff
@@ -5,27 +5,41 @@
     <title>IMDB Top 250</title>
```

Comment:
Really well done, Ronnie! This is some really good work. I just have a few comments on design and sharing data between controllers. We can discuss this after class on Monday or during office hours.